casacore/fitting/fitting.py

```diff
@@ -69,7 +68,7 @@ def fitter(self, n=0, ftype="real", colfac=1.0e-8, lmfac=1.0e-3):
             self._fitids.append({})
         else:
             # shouldn't happen
-            raise RangeError("fit id out of range")
+            raise IndexError("fit id out of range")
```

Comment:
Is this anything to do with Unicode support?

Reply:
To be honest, I don't remember. This PR has been open for soo long.